fastlane/helper/plugin_scores_helper.rb

```diff
@@ -0,0 +1,184 @@
+module Fastlane
+  module Helper
+    module PluginScoresHelper
+      require 'faraday'
+
+      class FastlanePluginRating
+        attr_accessor :key
+        attr_accessor :description
+        attr_accessor :value
+
+        def initialize(key: nil, description: nil, value: nil)
+          self.key = key
```

Comment:
this accesses the setter, right? wouldn't it be better to do `@key = key` etc...?